# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1490011976 -3600
#      Mon Mar 20 13:12:56 2017 +0100
# Branch stable
# Node ID c0cc56ab176d31d3095f8cf48032290431bb112e
# Parent  85ffe1bdd3a07963b06d7b802d0cb57359d34211
errorbar.m: force "box"->"on" like for other 2D high level plots

diff --git a/scripts/plot/draw/errorbar.m b/scripts/plot/draw/errorbar.m
--- a/scripts/plot/draw/errorbar.m
+++ b/scripts/plot/draw/errorbar.m
@@ -173,16 +173,20 @@ function h = errorbar (varargin)
 
   oldfig = [];
   if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
     hax = newplot (hax);
 
+    if (! ishold ())
+      set (hax, "box", "on");
+    endif
+    
     htmp = __errplot__ ("errorbar", hax, varargin{:});
   unwind_protect_cleanup
     if (! isempty (oldfig))
       set (0, "currentfigure", oldfig);
     endif
   end_unwind_protect
 
   if (nargout > 0)
